style.css

```diff
@@ -88,6 +89,341 @@ a {
   padding: 5px;
 }
 
+.categories-list li {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 3rem;
+  color: #fff;
+}
+
 .social-media ul li {
   padding: 5px;
 }
+
+#recentWorks {
+  display: grid;
+  height: auto;
+  grid-template-columns: 100%;
+}
+
+#recentWorks h2 {
+  color: #1c1a19;
+  font-weight: 400;
+  font-size: 36px;
+  border-bottom: 1px solid #212121;
+  align-items: center;
+  margin-top: 2em;
+  padding: 1em;
+}
+
+.imgHeader {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2vh;
+}
+
+.imgHeader img {
+  width: 340px;
+}
+
+#seeProjects {
+  position: relative;
+  height: 100%;
+}
+
+#works h3 {
+  color: #091e42;
+  font-size: 29px;
+  font-weight: 400;
+  font-family: 'Crete Round', sans-serif;
+  margin: 0;
+}
+
+.introInfo {
+  color: #091e42;
+  font-size: 15px;
+}
+
+#works {
+  margin: 0 11vw;
+}
+
+#works ul {
+  display: flex;
+  padding-left: 0;
+}
+
+#works ul li {
+  border: 1px solid #8993a4;
+  background: #fff;
+  padding: 10px 12px;
+  color: #091e42;
+  margin: 3px;
+  list-style-type: none;
+}
+
+#works button {
+  display: flex;
+  background: #ff6b00;
+  color: #fff;
+  align-items: center;
+  padding: 12px;
+  justify-content: center;
+  width: 110px;
+  margin-top: 3px;
+  border: none;
+  cursor: pointer;
+}
+
+#cardWork button {
+  width: 100%;
+  position: absolute;
+  background-color: #ff6b00;
+  padding: 15px 0;
+  text-align: center;
+  margin-top: 2.5vh;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  border: none;
+  bottom: 0;
+  cursor: pointer;
+}
+
+#contact-me button {
+  width: 40%;
+  background-color: #ff6b00;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  margin-top: 15px;
+  margin-bottom: 124px;
+  font-weight: 700;
+  font-size: 17px;
+  height: 48px;
+}
+
+#works button:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.prjectDescription {
+  padding-top: 11vh;
+  margin-left: 7px;
+}
+
+.bg {
+  background-image: url('./images/maskGroup.png');
+  position: relative;
+  width: 327px;
+  height: 356px;
+}
+
+#cardWork {
+  position: relative;
+  margin-bottom: 2vh;
+  left: 2em;
+  top: 2em;
+}
+
+#cardWork h3 {
+  font-size: 22px;
+  font-weight: 400;
+}
+
+#cardWork p {
+  font-size: 15px;
+  margin-right: 50px;
+}
+
+#cardWork ul {
+  display: flex;
+  padding-left: 0;
+  margin-bottom: 0;
+}
+
+#cardWork ul li {
+  display: flex;
+  background-color: #6b6b6b;
+  padding: 10px 12px;
+  justify-content: center;
+  align-items: center;
+  margin: 5px;
+  list-style-type: none;
+}
+
+#cardWork button:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.about-section {
+  background-color: rgba(0, 0, 0, 0.9);
+  font-family: 'Inter', sans-serif;
+  margin-top: 25vh;
```

Comment:
kindly avoid the use of vh since it behaves unpredicted on different screen sizes